tokio/src/runtime/mod.rs

```diff
@@ -312,6 +312,15 @@ cfg_rt! {
         blocking_pool: BlockingPool,
     }
 
+    /// The flavor of the `Runtime`.
+    #[derive(Debug, PartialEq)]
```

Comment:
This isn't a floating point number.
```suggestion
    #[derive(Debug, PartialEq, Eq)]
```

Reply:
Added.